sql/test_json.sh

```diff
@@ -86,9 +86,6 @@ else
     fail "COUNT_PATHS"
 fi
 
-echo "\"id555\", \"{\"\"c\"\":33}\"" | $SKDB load-csv t1
-echo "\"id555\", \"{\"\"c\"\":33}\"" | $SKDB load-csv t2
```

Comment:
We're missing the test data now? We should load the data in using an insert IMO, will be clearer.

It looks like below statements will fail, but none of the exit codes are checked.

Reply:
Is the current version what you had in mind?

Reply:
Yeah I think that looks good. I guess the test is happy now?

Reply:
Yes, but it doesn't mean much as I still have to figure out how to make the test unhappy when just deleting these lines...